library/src/SPSApplication.ts

```diff
@@ -61,7 +61,7 @@ export class SPSApplication extends Application {
 		this.stream.setSignallingUrlBuilder(() => {
 
 			// if we have overriden the signalling server URL with a .env file use it here
-			if (WEBSOCKET_URL) {
+			if (WEBSOCKET_URL != "") {
```

Comment:
Consider the case where `WEBSOCKET_URL` is `undefined`, e.g. in prod.

`undefined != ""` actually equals `true` - I don't think that is what you want here?

Reply:
This has been fixed up, 
It is now testing if the `WEBSOCKET_URL` is not `undefined`.
The only time the `WEBSOCKET_URL` var will be set is when building for `dev` and the `WEBSOCKET_URL` has been set in the `.env` file